src/models/spaces/cell-zoom/cell-zoom-row.ts

```diff
@@ -19,12 +19,17 @@ export type OrganelleType = typeof Organelle.Type;
 export const Mode = types.enumeration("type", ["add", "subtract", "assay", "normal"]);
 export type ModeType = typeof Mode.Type;
 
+export const ZoomLevel = types.enumeration("type", ["organism", "cell", "protein"]);
+export type ZoomLevelType = typeof ZoomLevel.Type;
```

Comment:
I added a `ZoomLevel` here because I thought I'd be implementing the organism view...it's not in use yet but I figured I'd leave it for reference.

Reply:
I think it's nice to have the enumeration. I don't know if it will cause conflicts when merging in with @dstrawberrygirl 's changes, but I think it's good to have the explicit words. If we eventually have multiple targets to zoom into, the model might even be able to set the options of this list.